### PR TITLE
Extend selectObjects with 'selectObjects:info' event

### PR DESCRIPTION
**Pull request by klinakerdisy**

### CODE REVIEW

**Review by klschm — COMMENTED** [review]
Looks good to me

**Review by jkissel — CHANGES_REQUESTED** [review]

File: src/cadenza.js [hunk 1]
```diff
@@ -957,6 +960,7 @@ function createParams({
 /** @typedef {CadenzaEvent<'selectObjects:update', {layer: WorkbookLayerPath, values: unknown[][]}>} CadenzaSelectObjectsUpdateEvent - When the user changed the selection. */
 /** @typedef {CadenzaEvent<'selectObjects:ok', {layer: WorkbookLayerPath, values: unknown[][]}>} CadenzaSelectObjectsOkEvent - When the user submitted the selection. */
 /** @typedef {CadenzaEvent<'selectObjects:cancel'>} CadenzaSelectObjectsCancelEvent - When the user cancelled the selection. */
+/** @typedef {CadenzaEvent<'selectObjects:info', {embeddingTargetId: EmbeddingTargetId, layer: WorkbookLayerPath, objectInfo: Record<string, unknown>[]}>} CadenzaSelectObjectsInfoEvent - When the user opened the object info flyout. */
```
[the remark this answers is not in the document]
Comment (klinakerdisy) [reply src/cadenza.js:963]: @jkissel What data to send is still under discussion and may still change. I will implement your suggestion regarding object info like we discussed on zoom.

[the remark this answers is not in the document]
Comment (klinakerdisy) [reply src/cadenza.js]: @jkissel The final changes to the event details have now been added.

File: src/cadenza.js [hunk 2]
```diff
@@ -957,6 +960,7 @@ function createParams({
 /** @typedef {CadenzaEvent<'selectObjects:update', {layer: WorkbookLayerPath, values: unknown[][]}>} CadenzaSelectObjectsUpdateEvent - When the user changed the selection. */
 /** @typedef {CadenzaEvent<'selectObjects:ok', {layer: WorkbookLayerPath, values: unknown[][]}>} CadenzaSelectObjectsOkEvent - When the user submitted the selection. */
 /** @typedef {CadenzaEvent<'selectObjects:cancel'>} CadenzaSelectObjectsCancelEvent - When the user cancelled the selection. */
+/** @typedef {CadenzaEvent<'objectInfo', {layer: WorkbookLayerPath, objectInfos: {selectedIndex: number, formattedValues: Record<string, string>}[]}>} CadenzaObjectInfoEvent - When the user opened the object info flyout. */
```
Comment (jkissel) [inline src/cadenza.js:963]: It's `selectionIndex` (the index in the workbook selection), right?

**Review by jkissel — CHANGES_REQUESTED** [review]

File: src/cadenza.js (hunk 2 above)
Comment (klinakerdisy) [reply src/cadenza.js]: Ah yes, that was a typo. Fixed it

**Pull request merged**